java-strings/src/test/java/com/baeldung/string/StringFromPrimitiveArrayUnitTest.java

```diff
@@ -0,0 +1,70 @@
+package com.baeldung.string;
+
+import com.google.common.base.Joiner;
+import com.google.common.primitives.Ints;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StringFromPrimitiveArrayUnitTest {
+
+    private static final int[] PRIMITIVE_ARRAY = { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+
+    private static final char SEPARATOR_CHAR = '-';
+
+    private static final String SEPARATOR = String.valueOf(SEPARATOR_CHAR);
+
+    private static final String EXPECTED_STRING = "1-2-3-4-5-6-7-8-9";
+
+    @Test
+    public void givenPrimitiveArray_whenJoinBySeparator_thenReturnsString_through_Java8CollectorsJoining() {
+        assertThat(Arrays.stream(PRIMITIVE_ARRAY)
+                .mapToObj(String::valueOf)
+                .collect(Collectors.joining(SEPARATOR))).isEqualTo(EXPECTED_STRING);
```

Comment:
Bring the ".isEqualTo" down to the next line.

Reply:
Done